config/routes.rb

```diff
@@ -5,4 +5,6 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users, only: [:show, :index]
+  resources :posts, only: [:show, :index]
```

Comment:
- [ ] Good job working on the controllers, but the route pattern does not match the project's route outline for the pages. For instance, I should be able to access the post index page from e.g `/users/1/posts`, instead of `/posts`.

Hint:
```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```